periph_sw/nRF52_RGB_Strip/led_lib.h

```diff
@@ -204,14 +210,32 @@ class CtrlLED
             // Switch on the leds.
             for (int i = leaderIdx; i < lastIdx; ++i)
             {
-                int ledIdx = i % this->numpixels;
+                int ledIdx = i % this->p_settings->num_pixels;
                 strip.setPixelColor(ledIdx, rgbiToColor(rgb.r, rgb.g, rgb.b, 0xFF));
             }
         }
 
         strip.show();
     }
 
+    void modeTraffic(void)
+    {
+        this->setPixelsOff(); // Init: switch evertyhing off.
+
+        // Front.
+        for (int i = std::max(this->p_settings->traffic_front_lower - 1, 0U); i < this->p_settings->traffic_front_upper; ++i)
+        {
+            strip.setPixelColor(i, strip.Color(255, 255, 255));
```

Comment:
A light yellow would be better. Pure white is generally too strong and aggressive, even at lower brightness settings.

Reply:
Agreed, will be changed later.